package.json

```diff
@@ -1,17 +1,35 @@
 {
   "name": "ultravox-client",
-  "version": "0.1.0",
+  "version": "0.2.9",
   "private": false,
+  "main": "./dist/cjs/session.js",
+  "module": "./dist/esm/session.js",
+  "browser": "./dist/esm/session.js",
+  "types": "./dist/esm/session.d.ts",
+  "exports": {
+    ".": {
```

Comment:
If we want to emit both ESM and CJS I strongly suggest using https://www.npmjs.com/package/tshy to manage this. Otherwise it's brutal trying to emit typescript properly (e.g. see [here](https://arethetypeswrong.github.io/?p=ultravox-client%400.2.9)).

Reply:
added tshy. pls check out the changes here in this PR